src/main/controllers/TribunalOrderOrRequestDetailsController.ts

```diff
@@ -35,6 +35,10 @@ export default class TribunalOrderOrRequestDetailsController {
     const respondButton = !selectedRequestOrOrder.value.respondCollection?.some(
       r => r.value.from === Applicant.CLAIMANT
     );
+    const responseRequired =
```

Comment:
Could this logic here be unit tested? might wanna have it as a function in a separate class as well

Reply:
This responseRequired variable check is to toggle the display of a paragraph which states "You must respond to the tribunal". Waiting on confirmation that this is okay or not, it wasn't part of the ticket.

Reply:
Added a unit test for this controller